floyd/cli/data.py

```diff
@@ -132,6 +134,7 @@ def output(id, url):
     Shows the url of the dataset. You can use id or a friendly URI.
```

Comment:
The id argument can now be optional.

Reply:
I combed through both the experiment and data commands to make sure ids are optional where they can be.

Reply:
👍